### ISSUE TITLE
README load example seems incorrect

### ISSUE THREAD

**Issue by terrycojones** [jonnyreeves/jquery-Mustache#10]
Judging from the code, it looks like this example from the README:

```
var viewData = { name: 'Jonny' };
$.Mustache.load('./templates/greetings.htm')
    .done(function () {
        $('body').mustache('simple-hello', viewData);
    });
});
```

should instead be

```
var viewData = { name: 'Jonny' };
$.Mustache.load('./templates/greetings.htm', function () {
        $('body').mustache('simple-hello', viewData);
});
```

**Comment by terrycojones** [comment]
Or maybe the `load` function should start out `return $.ajax(....)` ?

**Comment by jonnyreeves** [comment]
Hi Terry, thanks for raising this issue.  It would appear my last merge was a complete and total disaster!  I'll get this patched up and try and unpick the mess I caused :)

**Comment by terrycojones** [comment]
np :-)

**Comment by jonnyreeves** [comment]
Resolved in 0.2.4 - added test coverage and a build script in an attempt to ensure this sort of thing doesn't happen again :)
